pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -816,7 +814,18 @@ func getAppTopFolderFromPackage(rctx context.Context, cr splcommon.MetaObject, a
 	scopedLog.Info("Pod exec result", "stdOut", stdOut)
 
 	if stdErr != "" || err != nil {
-		return "", fmt.Errorf("could not get installed app name stdOut: %s, stdErr: %s, command: %s", stdOut, stdErr, command)
+		// CSPL-2598 - Log warnings/errors.
+		// Return an error only when an empty app name is extracted.
```

Comment:
We are able to and extract the name, we are erroring out because we are getting harmless warnings while doing so. 

Example log:

```
2024-04-30T12:56:06.22404625Z   INFO    podexec call returned   {"controller": "standalone", "controllerGroup": "enterprise.splunk.com", "controllerKind": "Standalone", "Standalone": {"name":"ido","namespace":"splunk-operator"}, "namespace": "splunk-operator", "name": "example", "reconcileID": "5676e6f1-41f3-43df-915d-a70bc0248fa4", "cmd": "/bin/sh", "stdout": "splunk_datasets_addon\n", "stderr": "tar: Ignoring unknown extended header keyword 'LIBARCHIVE.creationtime'\ntar: Ignoring unknown extended header keyword 'SCHILY.dev'\ntar: Ignoring unknown extended header keyword 'SCHILY.ino'\ntar: Ignoring unknown extended header keyword 'SCHILY.nlink'\ntar: Ignoring unknown extended header keyword 'LIBARCHIVE.creationtime'\ntar: Ignoring unknown extended header keyword 'SCHILY.dev'\ntar: Ignoring unknown extended header keyword 'SCHILY.ino'\ntar: Ignoring unknown extended header keyword 'SCHILY.nlink'\n", "err": ""}
```

Reply:
Successful installation of the app on a standalone.

![image](https://github.com/splunk/splunk-operator/assets/69369319/387cbb3a-37db-4abe-8f4d-a8816632e66b)